rvv-intrinsic-generator/rvv_intrinsic_gen/main.py

```diff
@@ -96,7 +98,7 @@ class GenTypes:
           "overloaded-docs", "non-overloaded-test", "overloaded-test",
           "non-overloaded-compatible-header", "overloaded-compatible-header"
       ])
-  parser.add_argument("--llvm", default=False, action="store_true")
+  parser.add_argument("--toolchain-type", default=ToolChainType.UNKNOWN)
```

Comment:
Need code to stop the generator if no `--toolchain-type` is specified.

Reply:
I tried  add the code in the generator.APITestGenerator.Funs function, but will affect the auto generated test cases at the auto-generated/API folder.  If we make sure that the toolchain-type specified in the Makefile is correct, can we not add the corresponding code?

Reply:
I tried  add the code in the generator.APITestGenerator.Funs function, but will affect the auto generated test cases at the auto-generated/API folder.  If we make sure that the toolchain-type specified in the Makefile is correct, can we not add the corresponding code?

Reply:
I mean, handle error under main.py to exit and emit error when `--toolchain-type` is not specified.

Reply:
OK! I got it. I will add code.

Reply:
Thanks for the reminder, we can still generate test case, like what we have right now in `api-testing/`, that don't need LLVM or GNU test headers. So I think no modifications is needed here.